src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -1087,8 +1101,7 @@ private int RunCore(TextWriter consoleOutput, ErrorLogger? errorLogger, Cancella
                     }
                 }
 
-                AnalyzerOptions analyzerOptions = CreateAnalyzerOptions(
-                      additionalTextFiles, analyzerConfigProvider);
+                AnalyzerOptions analyzerOptions = CreateAnalyzerOptions(additionalTextFiles, analyzerConfigProvider);
```

Comment:
Can we undo this? Looks like a style change only

Reply:
Fixed: https://github.com/dotnet/roslyn/pull/62641/files#diff-96f63d80d819d3bcedfa76e09251cb606e8d7b6da593d2457ed85566a882276d

Reply:
Fixed: https://github.com/dotnet/roslyn/pull/62641/files#diff-96f63d80d819d3bcedfa76e09251cb606e8d7b6da593d2457ed85566a882276d